src/android/com/adobe/phonegap/push/GCMIntentService.java

```diff
@@ -119,6 +122,49 @@ private void replaceKey(String oldKey, String newKey, Bundle extras, Bundle newE
     }
 
     /*
+     * Normalize localization for key
+     */
+    private String localizeKey(Context context, String key, String value) {
+        if (key.equals(TITLE) || key.equals(MESSAGE) || key.equals(SUMMARY_TEXT)) {
+            try {
+                JSONObject localeObject = new JSONObject(value);
```

Comment:
I disagree. It's a protective approach to not try to do string search out of context, as a full payload could have the string "lockey" somewhere without being necessarily a json key.